### DISCUSSION TITLE
2021-10-21 viewpoint culling

### DISCUSSION THREAD

**Discussion by alandtse** [alandtse/hdtSMP64#6] [Announcements]
Adds max angle parameter.

<hr /><em>This discussion was created from the release <a href='https://github.com/alandtse/hdtSMP64/releases/tag/2021-10-21_viewpoint_culling'>2021-10-21 viewpoint culling</a>.</em>

**Comment by BullOnMars** [comment]
Does viewport culling work on regular desktop verion?

**Reply by alandtse** [reply]
Unknown. My builds are for VR for now.

**Reply by alandtse** [reply]
Honestly, you can probably cherry pick the commit and it should work.
